plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -22,7 +22,7 @@ const config = {
 	testDir: 'tests',
 	retries: CI ? 2 : 0,
 	repeatEach: REPEAT_EACH ? Number( REPEAT_EACH ) : 1,
-	workers: CI ? 1 : 4,
+	workers: 1,
```

Comment:
Was this change intentional or maybe leftover from you testing? I would be ok with it either way, I usually disable it locally.

Reply:
It was intentional because on CI, we're running tests 1 at a time.  We used to be working towards 4 at a time everywhere, but since switching directions, it doesn't make sense to still run tests in parallel locally -- there's just too many failures.